Opening Changelog crashes program on macOS

Opening the Command pallete (⌘+Shift+P) and trying to open the changelog randomly crashes on macOS.

Running with `--debug` produces this:
```
[ERROR]    +2ms Oni2.Exception : Exception (Sys_error
  "/Users/zbaylin/Development/oni2/_esy/default/store/i/oni2-ecad36e8/bin/changelog.xml: No such file or directory"):
```

That file, in fact, does not exist, but running `find _esy -name "changelog.xml"` produces
```
_esy/default/store/b/oni2-ecad36e8/install/default/bin/changelog.xml
_esy/default/store/b/oni2-ecad36e8/default/assets/changelog.xml
```

This originates from here:
https://github.com/onivim/oni2/blob/f6d0d8e1568ea50e4d1608f8507340e2d4555596/src/Feature/Changelog/Feature_Changelog.re#L105-L107

I think it's a file permission bug or something with esy, but I'm not sure of that.

--- a/src/Feature/Changelog/Feature_Changelog.re
+++ b/src/Feature/Changelog/Feature_Changelog.re
@@ -30,6 +30,8 @@ let isSameDate = (a, b) => {
 // READ
 
 let read = () => {
+  open Base.Result.Let_syntax;
+
   let simplify = stream =>
     stream
     |> Markup.trim
@@ -103,8 +105,15 @@ let read = () => {
     | Text(_) => failwith("Unexpected text node");
 
   let path = Revery.Environment.getAssetPath("changelog.xml");
-  let simpleXml =
-    Markup.file(path) |> fst |> Markup.parse_xml |> Markup.signals |> simplify;
+
+  let%bind (stream, close) =
+    try(Markup.file(path) |> Result.ok) {
+    | exn => exn |> Printexc.to_string |> Result.error
+    };
+
+  let simpleXml = stream |> Markup.parse_xml |> Markup.signals |> simplify;
+
+  close();
 
   switch (parse(simpleXml)) {
   | commits => Ok(commits)

